lib/cdo/graphics/certificate_image.rb

```diff
@@ -234,7 +234,7 @@ def self.certificate_template_for(course)
       else
         'hour_of_code_certificate.jpg'
       end
-    elsif ScriptConstants.unit_in_category?(:twenty_hour, course)
+    elsif ScriptConstants.unit_in_category?(:twenty_hour, course) || course == 'accelerated'
```

Comment:
Should we leave a comment somewhere as to why we need to specifically check for 'accelerated' here?